src/Bokeh.jl

```diff
@@ -1,12 +1,15 @@
+VERSION >= v"0.4.0-dev+6641" && __precompile__()
```

Comment:
Sounds good to me.

We should tag a new minor release with current master to be our final 0.3 supported release.  That way if we have bugfixes to push back to 0.3 we can tag new patch releases based on that minor.